AsyncDisplayKit.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |spec|
       'Base/*.{h,m}'
   ]
 
-  spec.frameworks = 'AssetsLibrary'
+  spec.frameworks = 'AssetsLibrary','MapKit'
   spec.weak_frameworks = 'Photos'
```

Comment:
I don't actually know much about this, but can we make this a Weak framework?  @Adlai-Holler I think you'd set up the Photos one for us.  Curious if we can even do that with AssetsLibrary itself.